wrappers/java/src/test/java/org/hyperledger/indy/sdk/wallet/ExportWalletTest.java

```diff
@@ -0,0 +1,35 @@
+package org.hyperledger.indy.sdk.wallet;
+
+import org.hyperledger.indy.sdk.IOException;
+import org.hyperledger.indy.sdk.IndyIntegrationTestWithSingleWallet;
+import org.hyperledger.indy.sdk.did.Did;
+import org.junit.Test;
+
+import java.io.File;
+import java.util.concurrent.ExecutionException;
+
+import static org.hamcrest.CoreMatchers.isA;
+import static org.junit.Assert.assertTrue;
+
+
+public class ExportWalletTest extends IndyIntegrationTestWithSingleWallet {
```

Comment:
we should add a cleanup for created file(s)

Reply:
It cleanups the whole storage